packages/react-resizable-panels/src/PanelGroup.ts

```diff
@@ -123,7 +131,7 @@ function PanelGroupWithForwardedRef({
   forwardedRef,
   id: idFromProps = null,
   onLayout,
-  storage = defaultStorage,
+  storage = initializeDefaultStorage(),
```

Comment:
RE https://github.com/bvaughn/react-resizable-panels/pull/151#issuecomment-1559594826

> For instance, if we make a change like this– I think we may benefit from going one step further and deferring initialization until the first time storage is actually used (not just during props initialization). Maybe something like?

I think this still makes sense to do.